mne/minimum_norm/inverse.py

```diff
@@ -909,29 +945,35 @@ def apply_inverse_raw(raw, inverse_operator, lambda2, method="dSPM",
 
         # Allocate space for inverse solution
         n_times = data.shape[1]
-        sol = np.empty((K.shape[0] // 3, n_times),
+
+        n_dipoles = K.shape[0] if pick_ori == 'full' else K.shape[0] // 3
+        sol = np.empty((n_dipoles, n_times),
                        dtype=(K[0, 0] * data[0, 0]).dtype)
```

Comment:
we should really use `np.result_type` here but that can be done later if you want